.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "imgui-sys/third-party/imgui"]
-	path = imgui-sys/third-party/imgui
-	url = https://github.com/ocornut/imgui
+[submodule "imgui-sys/third-party/cimgui"]
```

Comment:
Unfortunately, I'm planning in moving in the other direction here. I'd rather not "double-down" on submodules for now, since I think for our use cases subtrees will be better (It also should help us use stuff like other branches long term).

So, for now I'd prefer if this update stuck with the way cimgui has been managed in the past in this repo, even though it is tedious.

Reply:
Yep fair enough, done!